shadow-cljs.edn

```diff
@@ -2,7 +2,11 @@
 {:source-paths ["src"]
  :dependencies [[re-frame "1.1.1"]
                 [re-graph "0.1.14"]
-                [metosin/reitit "0.5.10"]]
+                [metosin/reitit "0.5.10"]
+                [cljsjs/firebase "7.5.0-0"]
```

Comment:
I think that the cljsjs* packages are generally not used with shadow-cljs, is there a specific reason for this one?